internal/pkg/describe/env.go

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"encoding/json"
+	//"encoding/json"
+	"fmt"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+
+	//"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+)
+
+type Environment struct {
+	Name       string `json:"name"`
+	Production bool   `json:"production"` // add to instances used elsewhere (ie project_show)?
+	Region     string `json:"region"`
+	AccountID  string `json:"accountID"`
+}
+
+type EnvApp struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
```

Comment:
Is "Description" different from the `Type` field that is defined [here](https://github.com/aws/amazon-ecs-cli-v2/blob/master/internal/pkg/describe/project.go#L23)? If so we can just reuse the `Application` type instead of having a new `EnvApp` type.

Reply:
In the design meeting, there was talk of eventually having a 'description' field (@efekarakus ), to be populated by the user either upon init or in the manifest. That is currently out of scope, but for now should we call this field "Description" and just populate it with `Type`? Or call it `Type` until there's a real description?

Reply:
Let's do the latter (call it "Type").

Whenever we introduce a Description field to application manifests then we can add it to env show too.